spec/ssm_config_activerecord_spec.rb

```diff
@@ -1,36 +1,99 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
-
+require 'timecop'
 class SsmConfigDummy < ::ActiveRecord::Base
   self.table_name = 'ssm_config_records'
 end
 
 RSpec.describe SsmConfig do
   before do
-    stub_const('SsmConfig::CONFIG_PATH', '../fixtures')
-    stub_const('SsmConfig::ACTIVE_RECORD_MODEL', 'SsmConfigDummy')
+    stub_const('SsmStorageFile::CONFIG_PATH', '../fixtures')
+    stub_const('SsmStorageDb::ACTIVE_RECORD_MODEL', 'SsmConfigDummy')
     run_migrations(:up, migrations_path, 1)
+    SsmConfigDummy.new(:file => 'data', :accessor_keys => 'test,[0]', :value => 'hello').save
+    SsmConfigDummy.new(:file => 'data', :accessor_keys => 'other_key', :value => 'goodbye').save
+    Timecop.freeze(Time.zone.now)
+    Timecop.travel(Time.zone.now + 30.minutes)
   end
 
   let(:migrations_path) { SPEC_ROOT.join('support/active_record/postgres') }
   let(:error_message) { "undefined method `non_existent' for SsmConfig:Class" }
 
   after do
     run_migrations(:down, migrations_path)
+    described_class.instance_variable_set(:@data, nil)
   end
 
   context 'when table exists' do
+    it 'returns true' do
+      expect(ActiveRecord::Base.connection.table_exists?('ssm_config_records')).to eq(true)
```

Comment:
this isn't testing any of your/our logic. I think you want a `describe '#file_exists? do` block around some of these specs. The expectation then will always be `described_class.file_exists == false/true`.